src/PerfView/GcStats.cs

```diff
@@ -32,7 +32,9 @@ public static void ToHtml(TextWriter writer, TraceProcess stats, TraceLoadedDotN
                 writer.WriteLine("<LI><Font size=3 color=\"red\">Warning: Did not recognize the V4.0 GC Information events.  Falling back to V2.0 behavior.</font></LI>");
             if (!string.IsNullOrEmpty(stats.CommandLine))
                 writer.WriteLine("<LI>CommandLine: {0}</LI>", stats.CommandLine);
-            writer.WriteLine("<LI>Runtime Version: {0}</LI>", runtime.RuntimeVersion != null ? runtime.RuntimeVersion : "<Unknown Runtime Version>");
+            var runtimeBuiltTime = "";
+            if (runtime.RuntimeBuiltTime != default(DateTime)) runtimeBuiltTime = string.Format(" (built on {0})", runtime.RuntimeBuiltTime);
+            writer.WriteLine("<LI>Runtime Version: {0}{1}</LI>", runtime.RuntimeVersion != null ? runtime.RuntimeVersion : "<Unknown Runtime Version>", runtimeBuiltTime);
```

Comment:
@jeffschwMSFT Sorry about the confusion. The way I first read this, it looked like the following expression was added by you:

```csharp
runtime.RuntimeVersion != null ? runtime.RuntimeVersion : "<Unknown Runtime Version>"
```

My suggestion (:bulb:) gives a shorter expression which is equivalent to it, while the ❗️ points out that if the fallback string is ever used then the output is invalid. Even though you didn't add this code originally, it would be good to correct at least the latter issue while you're on the same line as it by changing `<`&rarr;`&lt;` and `>`&rarr;`&gt;`. Otherwise I'll file a separate issue for this.